pv211_utils/preprocessing/math_preprocessing.py

```diff
@@ -0,0 +1,17 @@
+from sympy import sympify, latex
```

Comment:
@MarekToma: Like with [`datasets.py`][1], use the classes from this module in the Jupyter notebook `notebooks/arqmath.ipynb`. The free runtime provided by Google Colab will be insufficient. Use [the PV211 JupyterHub][1] instead.

 [1]: https://github.com/MIR-MU/pv211-utils/pull/8/files/e5d5d7da1e36682fdcb11ddd1efafc727fc97109#r1124185893

Reply:
I'm not sure if we understand each other correctly. The math_preprocessing module itself was designed to help with converting plain math expressions to MathML/Latex, so right now, we are not even able to work with ARQMath Dataset.